README.md

```diff
@@ -30,13 +34,13 @@ If you'd like to run locust on its own, you can use the following commands
 For the web interface:
 
 ```bash
-locust -f runner_benchmark/har_replay_test.py --host=http://localhost:5000
+EQ_ID=test FORM_TYPE=mutually_exclusive locust -f runner_benchmark/har_replay_test.py --host=http://localhost:5000
```

Comment:
Should mention in the readme that these need to be run inside your virtual env, since the previous command does `pipenv run`.